docs/advanced-guide/_category_.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
can we keep a single json file in website folder for this instead of adding a separate json file in every docs folder?

Reply:
also, can we try to add sections in the same json only instead of separating doc md files in folders?